Makefile

```diff
@@ -0,0 +1,20 @@
+# Sane Makefile Preamble from https://tech.davis-hansson.com/p/make/
```

Comment:
Why is this whole thing even a Makefile instead of just a shell/Python/whatever script? There seems to be nothing that make adds here other than a lot of effort to force it to act as close as possible to a bash script?

Note that we need to invoke this from `etc/update.sh`. So how about turning this into, say, a script `etc/fetch_examples_status.sh` which then is invoked from `update.sh` ? Or even directly make it a part of `update.sh`.

Reply:
I forgot about the `update.sh` script, and a Makefile is my default answer for "we need to do this preprocessing step before we invoke build" so whoever is using the project just needs to run `make`.

I have removed the makefile and moved things to the update script.